packages/create-agents/README.md

```diff
@@ -0,0 +1,163 @@
+# create-agents
+
+Create an Inkeep Agent Framework project with no build configuration.
+
+## Quick Start
+
+```bash
+# Interactive mode
+npm init agents
+# or
+npx create-agents
+
+# With project name
```

Comment:
I don't want to mix the concepts of repos and Agent Framework object `Project`. Can we make sure these are separate concepts? Can you think of a clear way of doing it?